recipes/embree/all/conanfile.py

```diff
@@ -0,0 +1,106 @@
+
+import os
+from conans import CMake, tools
+from conans.model.conan_file import ConanFile
+
+
+class Embree(ConanFile):
+    name = "embree"
+    license = "Apache-2.0"
+    url = "https://github.com/conan-io/conan-center-index"
+    topics = ("conan", "embree", "raytracing", "rendering")
+    description = "Intel's collection of high-performance ray tracing kernels."
+    generators = "cmake"
```

Comment:
```suggestion
    generators = "cmake", "cmake_find_package"
```
fix macos tbb find error

Reply:
added